### PR TITLE
chore: add e2e tests, cleanup dummy unit tests

### PR DESCRIPTION
**Pull request by adrkakol**

### CODE REVIEW

File: src/helpers/get-auth-header-from-context-request.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { IncomingMessage } from 'http';
```
Comment (adrkakol) [inline src/helpers/get-auth-header-from-context-request.spec.ts:1]: unit test - take a look where the file is placed

File: src/helpers/get-auth-header-from-context-request.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { IncomingMessage } from 'http';
 
 export const getAuthHeaderFromContextRequest = (request: IncomingMessage): string | null => {
   const authHeader = request?.headers?.authorization || null;
-  const hasAuthHeader = authHeader && authHeader.length > 0 && authHeader.indexOf('Bearer ') > -1;
+  const hasAuthHeader = authHeader && authHeader.length > 0 && authHeader.indexOf('Bearer ') === 0;
```
Comment (adrkakol) [inline src/helpers/get-auth-header-from-context-request.ts:5]: Fixed and tested

File: src/tests/test-helpers/database-helper.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { TestingModule } from '@nestjs/testing';
+import { getModelToken } from '@nestjs/sequelize';
+
+import { Place } from '../../places/models/place.model';
+
+export class DatabaseHelper {
+  constructor(private readonly testingModule: TestingModule) {}
+
+  public readonly placeRepository = this.testingModule.get<typeof Place>(getModelToken(Place));
```
Comment (adrkakol) [inline src/tests/test-helpers/database-helper.ts:9]: this is how to add more db handlers for creating entities in our e2e tests

**Review by michaldomagalski — APPROVED** [review]
Nice. I would just maybe go with pluralized directories names for tests, i.e. `controllers` as we have this convention in the code.

**Pull request merged**